stories/documentation/navigation/segmentedControl/segmentedControl-filter.stories.ts

```diff
@@ -0,0 +1,74 @@
+import { Meta, StoryFn } from '@storybook/angular';
+
+interface segmentedControlFilterStory {
+	S: string;
+	withNumericBadge: boolean;
+	vertical: boolean;
+}
+
+export default {
+	title: 'Documentation/Navigation/segmentedControl/Filter',
+	argTypes: {
+		S: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		vertical: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		withNumericBadge: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: segmentedControlFilterStory): string {
+	const size = args.S ? `mod-S` : '';
+	const vertical = args.vertical ? `mod-vertical` : '';
+	return `<ul class="segmentedControl ${size} ${vertical}" role="presentation">
+	<li class="segmentedControl-item">
```

Comment:
here we don't need role presentation too ?

Reply:
I don't think the number of items in the list will be of any interest, since we're in form navigation mode.

Reply:
i agree that's why i propose to add a role presentation on `<li>` too

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role#description

> When an element has required descendants, such as the various `<table>`, elements and `<li>`s children of a `<ul>` or `<ol>`, the presentation or none role on the table or list removes the default semantics of the element on which it was applied and their required descendant elements.

…which is very convenient!

Reply:
Labels and inputs are also descendant. Won't the parent role="presentation" remove their semantic too ?
Why using an unordered list markup and then remove it's semantic ? Maybe juste use span in that case ?

Reply:
Yes, it depends on the HTML structure and will stop at the content.

> If presentation or none is applied to a `<table>` element, the descendant `<caption>`, `<thead>`, `<tbody>`, `<tfoot>`, `<tr>`, `<th>`, and `<td>` elements inherit the role and are thus not exposed to assistive technologies. But, elements inside of the `<th>` and `<td>` elements, including nested tables, are exposed to assistive technologies.

This gives us two possible structures:

- `ul role="presentation"`
- `li role="presentation"`(inherited)
- `input`

- `ul role="tablist"`
- `li role="presentation"`
- `button role="tab”`

The UL structure is completely optional, but I find it a bit structuring nonetheless.

Reply:
Shouldn't the version with the `input:radio` be contained in a `fieldset` rather than a list?

Reply:
Good point! But come to think of it, it would only make sense if we had a common label on display.
For the moment, that sounds good to me.

Reply:
I think we will probably need a hidden legend for better a11y as they generally be used to filter a list:

`All | Status 1 | Status 2`

Reply:
I'm adding this to my thoughts for the next release.